UI/Pages/Enquiry/Build/SchoolPostcode.cshtml.cs

```diff
@@ -0,0 +1,52 @@
+using Application.Common.Interfaces;
+using Application.Common.Models.Enquiry.Build;
+using Domain;
+
+namespace UI.Pages.Enquiry.Build;
+
+public class SchoolPostcode : PageModel
+{
+    private readonly ISessionService _sessionService;
```

Comment:
Field '_sessionService' is assigned but its value is never used. We can remove this.

Reply:
Done